cli/cycpp.py

```diff
@@ -112,6 +113,41 @@
 ENTITIES = [('cyclus::Region', 'region'), ('cyclus::Institution', 'institution'), 
             ('cyclus::Facility', 'facility'), ('cyclus::Agent', 'archetype')]
 
+def escape_xml(s, ind='    '):
+    """Escapes xml string s, prettifies it and puts in c++ string lit form."""
+
+    s = xml.dom.minidom.parseString(s)
+    s = s.toprettyxml(indent='    ')
+    s = s.replace('"', '\\"')
+
+    lines = s.splitlines()
+    lines = lines[1:] # remove initial xml version tag
+
+    clean = []
+    for line in lines:
+        if line.strip() != '':
+            clean.append(line)
```

Comment:
This should be a list comprehension and the predicate should be `len(line.strip()) == 0`